### PR TITLE
[Oslo-2023] Add new event

### PR DESCRIPTION
**Pull request by larstobi**
Added Oslo 2023 event!

### CODE REVIEW

File: data/events/2023-oslo.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+city: "Oslo" # The displayed city name of the event. Capitalize it.
+event_twitter: "devopsdaysoslo" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devopsdays is coming to Oslo!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2023-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2023-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
```
Comment (MikeRosTX) [inline data/events/2023-oslo.yml:12]: **_startdate_** & **_enddate_** need to be removed until you have a confirmed signed contract for a **_location_**.

File: data/events/2023-oslo.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+ # - name: speakers
+  - name: sponsor
+  - name: contact
+  - name: conduct
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
```
Comment (MikeRosTX) [inline data/events/2023-oslo.yml:50]: Images for organisers from 2022 seem to be missing for 2023.

**Review by MikeRosTX — CHANGES_REQUESTED** [review]
Make changes requested to **_2023-oslo.yml_**.
Oslo email dl and slack have been updated.

File: data/events/2023-oslo.yml (hunk 1 above)
Comment (larstobi) [reply data/events/2023-oslo.yml:12]: We already have a signed contract for a location with Sentralen, Oslo.

File: data/events/2023-oslo.yml (hunk 2 above)
Comment (larstobi) [reply data/events/2023-oslo.yml:50]: I have now added the organizer images, except for two new organizers who haven't provided images yet.

Comment (larstobi) [reply data/events/2023-oslo.yml:50]: I have now added the pictures for the two new organizers too.

**Pull request merged**